### PR TITLE
Fix/arborist

### PR DESCRIPTION
**Pull request by philloooo**
- without the networkpolicy it still works in dev, so we should fix the dev environment to respect networkpolicy so testing in dev is more useful
- added some doc

### CODE REVIEW

File: kube/services/netpolicy/networkpolicy_arborist_templ.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            app: arborist
+      - podSelector:
+          matchLabels:
+            app: fence
+      - ipBlock:
+           cidr: 0.0.0.0/0
+      - ipBlock:
+           cidr: 54.224.0.0/12
+      - ipBlock:
+           cidr: 52.92.16.0/20
```
Comment (frickjack) [inline kube/services/netpolicy/networkpolicy_arborist_templ.yaml]: Does arborist need egress to the internet?

Comment (philloooo) [reply kube/services/netpolicy/networkpolicy_arborist_templ.yaml]: oh I guess not yet

**Comment by frickjack** [comment]
you need to deploy some network policy to your dev environment for it to take affect: `gen3 kube-setup-networkpolicy` or `gen3 roll all` should do it too - if no network policies have been deployed, then the cluster assumes open communication

**Comment by frickjack** [comment]
Ugh - maybe I spoke too soon - might need to deploy Calico on the cluster - I'll talk with Fauzi:
   https://docs.aws.amazon.com/eks/latest/userguide/calico.html

**Comment by philloooo** [comment]
Yeah it was a new cluster so I rolled all

**Pull request merged**